ncs/Kconfig

```diff
@@ -26,6 +47,7 @@ config SUIT_ENVELOPE_SEQUENCE_NUM
 config SUIT_ENVELOPE_DEFAULT_TEMPLATE
     string "Path to the default envelope template (deprecated)"
     default "${ZEPHYR_SUIT_GENERATOR_MODULE_DIR}/ncs/root_with_nordic_top_envelope.yaml.jinja2" if SOC_NRF54H20_CPUAPP
+    default "${ZEPHYR_SUIT_GENERATOR_MODULE_DIR}/ncs/root_with_binary_nordic_top.yaml.jinja2" if SOC_NRF54H20_ENGA_CPUAPP
```

Comment:
You probably may use `INCLUDE_SECDOM` instead

Reply:
Switching to `INCLUDE_SECDOM` causing some problems while building SDFW and sysctrl projects.
If we would like to use the following condition:
```
default "${ZEPHYR_SUIT_GENERATOR_MODULE_DIR}/ncs/root_with_binary_nordic_top.yaml.jinja2" if !INCLUDE_SECDOM
```

Then condition is met in the SDFW/sysctrl project and wrong template is used.

If we would like to go with `INCLUDE_SECDOM` then we need to define much more complex condition.